### PR TITLE
AER-2564 Add validation warning on volume sources that seems to float.

### PR DESCRIPTION
**Pull request by Hilbrand**

### CODE REVIEW

File: source/imaer-util/src/main/java/nl/overheid/aerius/validation/ADMSCharacteristicsValidator.java [hunk 1]
```diff
@@ -39,9 +39,11 @@ boolean validate(final ADMSSourceCharacteristics characteristics) {
     if (characteristics.getSourceType() != SourceType.VOLUME && characteristics.getSourceType() != SourceType.ROAD) {
       valid = validateADMSHeatContent(characteristics);
     }
+    validateADMSVolumeHeight(characteristics);
     return valid;
   }
 
+
```
Comment (BertScholten) [inline source/imaer-util/src/main/java/nl/overheid/aerius/validation/ADMSCharacteristicsValidator.java:46]:
```suggestion
```

File: source/imaer-util/src/test/java/nl/overheid/aerius/validation/ADMSCharacteristicsValidatorTest.java [hunk 2]
```diff
@@ -102,4 +102,35 @@ private static Stream<Arguments> casesForADMSHeatCapacity() {
...
+      assertEquals(1, warnings.size(), "Number of warning");
+      assertEquals(ImaerExceptionReason.SOURCE_VOLUME_FLOATING, warnings.get(0).getReason(), "Expected source volume floating warning.");
+    } else {
+      assertTrue(warnings.isEmpty(), "Didn't expect any warnings");
+    }
+  }
+
+  private static Stream<Arguments> casesForADMSVerticalDimensionHeight() {
+    return Stream.of(
+        //           source type, height, vertical dimension, warning
```
Comment (BertScholten) [inline source/imaer-util/src/test/java/nl/overheid/aerius/validation/ADMSCharacteristicsValidatorTest.java:130]: The extra spaces are added to align with the arguments?

**Review by BertScholten — APPROVED** [review]
LGTM

File: source/imaer-util/src/test/java/nl/overheid/aerius/validation/ADMSCharacteristicsValidatorTest.java (hunk 2 above)
Comment (Hilbrand) [reply source/imaer-util/src/test/java/nl/overheid/aerius/validation/ADMSCharacteristicsValidatorTest.java:130]: Yes these spaces are an attempt to align with the arguments.

**Pull request merged**